jupyter-extension/setup.cfg

```diff
@@ -18,6 +18,7 @@ classifiers =
 [options]
 packages = find:
 install_requires =
+  pyyaml>=6.0
```

Comment:
I don't think we need to change this, but just to note: in my experience `ruamel.yaml` is a more complete yaml parser than `pyyaml`. Since out input specs use basic yaml I don't think its an issue. Here's a reference for the differences: https://yaml.readthedocs.io/en/latest/pyyaml.html